operator/pkg/values/translation/translation_test.go

```diff
@@ -412,14 +417,63 @@ func TestSetScannerV4ComponentValues(t *testing.T) {
 
 func TestSetScannerV4DBValues(t *testing.T) {
 	tests := map[string]struct {
-		db      *platform.ScannerV4DB
-		want    chartutil.Values
-		wantErr bool
+		db          *platform.ScannerV4DB
+		want        chartutil.Values
+		fakeObjects []runtime.Object
+		kind        string
+		wantErr     bool
 	}{
-		"empty for default component": {
+		"empty for central default component": {
+			db:   &platform.ScannerV4DB{},
+			kind: platform.CentralGVK.Kind,
+			want: chartutil.Values{},
+		},
+		"empty for securedcluster default with default StorageClass": {
 			db:   &platform.ScannerV4DB{},
+			kind: platform.SecuredClusterGVK.Kind,
+			fakeObjects: []runtime.Object{
+				&storagev1.StorageClass{
+					ObjectMeta: v1.ObjectMeta{
+						Name: "test-sc1",
+					},
+				},
+				&storagev1.StorageClass{
+					ObjectMeta: v1.ObjectMeta{
+						Name: "test-sc2",
+						Annotations: map[string]string{
+							DefaultStorageClassAnnotationKey: "true",
+						},
+					},
+				},
+			},
 			want: chartutil.Values{},
```

Comment:
Sorry for using you as my rubber duck here...

I'm not sure if the output values should be empty in this case.
Assumption is we have a default SC.
The Helm chart will be rendered without any persistence configuration.
Hence the defaulting within the Helm chart will kick in.
We are rendering for a secured cluster.
Storage classes are looked up.
But lookups are actually disabled, so the lookup will not return anything.
This should be the relevant code path: https://github.com/stackrox/stackrox/blob/3412bfff6bca2f592feac788495ac5a938b5c277/image/templates/helm/shared/templates/_scanner-v4_volume.tpl#L56
Hence, `emptyDir` would be used. no?

I would expect that we set something like
```
scannerV4.db.persistence.persistentVolumeClaim = { "createClaim": true}
```

Reply:
As discussed in our meeting this was a bug, good catch thank you. Fixed that in my last commit.